.github/CODEOWNERS

```diff
@@ -0,0 +1,34 @@
+# GitHub CODEOWNERS definition
+# See: https://help.github.com/articles/about-codeowners/
+
+* @elastic/beats
+
+/auditbeat/module/ @elastic/secops
+/x-pack/auditbeat/ @elastic/secops
+
+/packetbeat/protos/ @elastic/secops
+/x-pack/packetbeat/ @elastic/secops
+
+/metricbeat/module/ @elastic/infrastructure
+/metricbeat/modules.d/ @elastic/infrastructure
+/metricbeat/module/elasticsearch/ @elastic/stack-monitoring
+/metricbeat/module/kibana/ @elastic/stack-monitoring
+/metricbeat/module/logstash/ @elastic/stack-monitoring
+/x-pack/metricbeat/module/ @elastic/infrastructure
+/x-pack/metricbeat/modules.d/ @elastic/infrastructure
+
+/filebeat/module/ @elastic/infrastructure
+/filebeat/modules.d/ @elastic/infrastructure
+/filebeat/module/elasticsearch/ @elastic/stack-monitoring
+/filebeat/module/kibana/ @elastic/stack-monitoring
+/filebeat/module/logstash/ @elastic/stack-monitoring
+/x-pack/filebeat/module/ @elastic/infrastructure
+/x-pack/filebeat/modules.d/ @elastic/infrastructure
+/x-pack/filebeat/module/suricata/ @elastic/secops
+/x-pack/filebeat/modules.d/suricata.yml.disabled @elastic/secops
+
+/heartbeat/ @elastic/uptime
+
+/journalbeat/module/ @elastic/infrastructure
+/x-pack/journalbeat/ @elastic/infrastructure
+
```

Comment:
For elastic/beats, I think we are OK with that change because we will receive PR review request for any code created in beats because of the * @elastic/beats and because creating a module often creates files outside of the module directory AKA yml config and fields changes.

Reply:
Sorry, my mistake of adding a special case for journalbeat.